.circleci/config.yml

```diff
@@ -817,21 +817,19 @@ jobs:
           version: "<< pipeline.parameters.docker-engine-version>>"
       - run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin
 
-      - run: core/files_artifacts_expander/scripts/build.sh
+      - run: core/files_artifacts_expander/scripts/build.sh true
```

Comment:
cool we are doing this so that we aren't building the image multiple times

Reply:
Exactly 👍